tests/functional/vault/test_withdrawal.py

```diff
@@ -306,7 +306,7 @@ def test_profit_degration(chain, gov, token, vault, strategy, rando):
 
     pricePerShareBefore = vault.pricePerShare()
 
-    chain.sleep(10)
+    chain.sleep(1_000)
```

Comment:
Why'd this change?

Reply:
The value isn't changing enough if you use a token with only two decimal so vault.pricePerShare()  == pricePerShareBefore

Reply:
was it not passing? or was it not precise enough to show any difference?

Reply:
It's not passing because of the assert a line after

Reply:
Can you explain in greater detail? It sounds like you modified a test to get it to pass. What am I missing about why this change is justifiable?

Reply:
```
@view
@internal
def _shareValue(shares: uint256) -> uint256:
    # Determines the current value of `shares`.
        # NOTE: if sqrt(Vault.totalAssets()) >>> 1e39, this could potentially revert
    lockedFundsRatio: uint256 = (block.timestamp - self.lastReport) * self.lockedProfitDegration
    freeFunds: uint256 = self._totalAssets()

    if(lockedFundsRatio < DEGREDATION_COEFFICIENT):
        freeFunds -= (self.lockedProfit - (lockedFundsRatio * self.lockedProfit / DEGREDATION_COEFFICIENT))
    # NOTE: using 1e3 for extra precision here, when decimals is low
    return ((10 ** 3 * (shares * freeFunds)) / self.totalSupply) / 10 ** 3

```

Constants:
```
DEGREDATION_COEFFICIENT = 10 ** 18
lockedProfitDegration = DEGREDATION_COEFFICIENT * 46 /10 ** 6 = 4.6e+13
```
At the moment of the `    pricePerShareBefore = vault.pricePerShare()` is called at least a second has passed since harvest was called.

```
lockedFundsRatio = 1 * 4.6e+13 = 4.6e+13
freeFunds = 100_000
lockedProfit = 10

(10 - (4.6e+13 * 10 / 10 ** 18)) = 9.99954
Because of the rounding we get:
freeFunds -= 9
```

And then ten seconds after the withdrawal we have:
```
lockedFundsRatio = 10 * 4.6e+13 = 4.6e+14
freeFunds = 100_000
lockedProfit = 10

(10 - (4.6e+14 * 10 / 10 ** 18)) = 9.9954
Because of the rounding we get:
freeFunds -= 9
```

So no more funds are freed between the two calls.

Reply:
So to summarize, it's just not waiting long enough to see an effect when `decimals` is a lower number? e.g. There is no losses occurring, just not enough time to see gains

Reply:
There is not enough time to see  difference because of the rounding of the lock degradation.